debs/bionic/fits/debian/control

```diff
@@ -2,13 +2,12 @@ Source: fits
 Section: utils
 Priority: extra
 Maintainer: austin trask <austin@artefactual.com>
-Build-Depends: debhelper (>= 7), default-jdk, ant-gcj
+Build-Depends: debhelper (>= 7), openjdk-8-jdk-headless, ant-gcj
 Standards-Version: 3.8.3
 Homepage: http://code.google.com/p/fits/
 
 Package: fits
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, nailgun (>= 0.9), default-jdk
```

Comment:
Depends: Should be `openjdk-11-jre-headless` instead of `default-jdk`.

Reply:
I left `openjdk-8-jdk-headless` as build dependency, and set `openjdk-8-jre-headless` as the run dependency